src/vorta/utils.py

```diff
@@ -268,7 +266,22 @@ def find_best_unit_for_size(size: Optional[int], metric: bool = True, precision:
     return n
 
 
-def pretty_bytes(
+def pretty_bytes_dynamic_units(size, metric=True, sign=False, precision=1):
+    if not isinstance(size, int):
+        return ''
+    prefix = '+' if sign and size > 0 else ''
+    power, units = (10**3, METRIC_UNITS) if metric else (2**10, NONMETRIC_UNITS)
+    n = find_best_unit_for_size(size, metric=metric, precision=precision)
+    size /= power**n
+    try:
+        unit = units[n]
+        return f'{prefix}{round(size, precision)} {unit}B'
+    except KeyError as e:
+        logger.error(e)
+        return "NaN"
+
```

Comment:
Now that you changed this method to use `find_best_unit_for_size` it does exactly the same as `pretty_bytest_fixed_units` when passing `fixed_unit=None`. Consequently we only need the latter.

Reply:
I just combined these two into one `pretty_bytes()` function, and rewrote the tests to ensure functionality.

Reply:
Also, went through the files affected by previous (and now reverted) `pretty_bytes` name change and set them back to their original state.